### ISSUE TITLE
Homescreen icon on IOS

### ISSUE THREAD

**Issue by TribuneX** [BookStackApp/BookStack#301]
Desired Feature: Support for Homescreen icons on iOS: https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariWebContent/ConfiguringWebApplications/ConfiguringWebApplications.html

**Comment by btone-comm** [comment]
This is achieved through the settings page, and placing the images in your BookStack public folder.



Add the following code on the Settings page, under "Custom HTML head content" section.
```html
<link rel="apple-touch-icon" sizes="57x57" href="/apple-icons/apple-touch-icon-57x57.png" />
<link rel="apple-touch-icon" sizes="72x72" href="/apple-icons/apple-touch-icon-72x72.png" />
<link rel="apple-touch-icon" sizes="72x72" href="/apple-icons/apple-touch-icon-76x76.png" />
<link rel="apple-touch-icon" sizes="114x114" href="/apple-icons/apple-touch-icon-114x114.png" />
<link rel="apple-touch-icon" sizes="144x144" href="/apple-icons/apple-touch-icon-144x144.png" />
<link rel="apple-touch-icon" sizes="144x144" href="/apple-icons/apple-touch-icon-120x120.png" />
<link rel="apple-touch-icon" sizes="144x144" href="/apple-icons/apple-touch-icon-152x152.png" />
<link rel="apple-touch-icon" sizes="144x144" href="/apple-icons/apple-touch-icon-180x180.png" />
```

Place your icons in you BookStack public folder, in a folder named apple-icons. 
`\public\apple-icons`

You can generate your icons with a site, like: [http://iconifier.net/](http://iconifier.net/)

**Comment by ssddanbrown** [comment]
I've just merged in  #3994 which, from my testing on iPad, allows setting an app icon that will be used as the "Add to homescreen" icon. Will be part of the next release.
